pyproject.toml

```diff
@@ -113,7 +113,7 @@ netaddr = "~0.8.0"
 #       we permit a looser (^) version constraint here.
 netutils = "^1.6.0"
 # Handling of version numbers
-packaging = "~23.1"
+packaging = "^23.2"
```

Comment:
I'm a bit hesitant over raising the minimum version here as we've had doing so for `packaging` specifically cause conflicts with downstream apps before. Also, I see that `packaging` uses CalVer rather than SemVer, so this will still prevent installation of the next minor release of `packaging` since that will be a `24.x` version number at this point even if there are no breaking changes.

Taking all of that into account, I might suggest  that we just go with `poetry add 'packaging>=23.1'` for now, and *hope* that that package doesn't release any breaking changes.

Reply:
That seems reasonable to me.

Reply:
@glennmatthews I'm reconsidering whether this is actually an issue. The original "issue" is `packaging` version between pynautobot and nautobot... One could simply run them from different environments rather than installing them in the same environment. Thoughts?

Reply:
In general that would be the more common scenario, but I can also see that there might be developer use cases where you want to have both installed and not have to manage two different venvs.